components/ModSubmissionsEditFormFacility.vue

```diff
@@ -15,6 +15,7 @@
                 :required="true"
                 :input-validation-check="validateNameEn"
                 :invalid-input-error-message="$t('modSubmissionForm.inputErrorMessageFacilityNameEn')"
+                :autofill-value="moderationInputStore.nameEn"
```

Comment:
I might be missing something... do we need an autofill prop? Can't we just set the actual value when the parent data loads?

Reply:
We are doing that in the `ModInput` component. But the value is different and this is a reused component. Otherwise it would set all the inputs to value? No?

Reply:
We are setting the initial value in the `ModInput` when the parent data loads.

```
const setInitialValue = () => {
    if (props.autofillValue) {
        inputValue.value = props.autofillValue
    }
}
onMounted(() => [
    setInitialValue()
])
```

Reply:
I tried removing it but it doesn't override the placeholder in each component even if the value in the store is set to the autofill value.